src/main/java/edu/usf/cutr/grha/utils/ChicagoDataUtils.kt

```diff
@@ -0,0 +1,16 @@
+package edu.usf.cutr.grha.utils
+
+import edu.usf.cutr.grha.model.ChicagoData
+
+object ChicagoDataUtils {
+    @JvmStatic
+    fun printChicagoData(data: List<ChicagoData>) {
+        for (d in data) {
```

Comment:
I think you could just print `data.toString()` here instead of printing it all our yourself?

Reply:
I would but printing this way, it looks much nicer and easier to read on the terminal. If I use `toString()`, it looks like something goes really wide on the terminal for bigger datasets like Chicago Data.